src/index.html

```diff
@@ -41,9 +41,27 @@ <h1>Github Repository Control</h1>
                         </tr>
                     </tbody>
                 </table>
-                <button class="btn btn-danger btn-sm my-2 text-uppercase" v-on:click="delete_repository">delete repositories</button>
+                <button type="button" class="btn btn-danger btn-sm my-2 text-uppercase" data-bs-toggle="modal" data-bs-target="#confirm_action_modal">delete repositories</button>
             </div>
         </div>
+            <!-- Modal -->
+            <div class="modal fade" id="confirm_action_modal" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1" aria-labelledby="confirm_action_modal_label" aria-hidden="true">
+                <div class="modal-dialog">
+                <div class="modal-content">
+                    <div class="modal-header">
+                    <h1 class="modal-title fs-5" id="confirm_action_modal_label">Confirm Action</h1>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                    </div>
+                    <div class="modal-body">
+                    Are you sure you want to delete the selected repositories?
```

Comment:
Indenting this would make it easier to see it is inside of div.modal-body

Reply:
I will format HTML